src/us/kbase/workspace/test/kbase/HandleAndBytestreamIntegrationTest.java

```diff
@@ -598,10 +594,12 @@ public void saveAndGetWithBytestreamIDs() throws Exception {
 			System.out.println(se.getData());
 			throw se;
 		}
+		assertThat("incorrect owner", n1.getACLs().getOwner(), is(SHOCK_USER3));
 		checkReadAcl(n1, Arrays.asList(SHOCK_USER3, SHOCK_USER1));
 		checkWriteAcl(n1, Arrays.asList(SHOCK_USER3));
 		checkDeleteAcl(n1, Arrays.asList(SHOCK_USER3));
-		checkReadAcl(n2, Arrays.asList(SHOCK_USER1));
+		assertThat("incorrect owner", n2.getACLs().getOwner(), is(SHOCK_USER3));
+		checkReadAcl(n2, Arrays.asList(SHOCK_USER3, SHOCK_USER1));
```

Comment:
When a node's owner is changed by the blobstore, it adds the owner to the read ACL as well. This is a change from Shock.